app/_src/installation/docker.md

```diff
@@ -22,9 +22,9 @@ The official Docker images are used by default in the [Kubernetes](/docs/{{ page
 
 You can freely `docker pull` these images to start using {{site.mesh_product_name}}, as we will demonstrate in the following steps.
 
-### 2. Run {{site.mesh_product_name}}
+## Run {{site.mesh_product_name}}
 
-We can run {{site.mesh_product_name}}:
+You can run {{site.mesh_product_name}}:
 
 `docker run -p 5681:5681 docker.io/kumahq/kuma-cp:{{ page.latest_version }} run`
```

Comment:
Since these installation files were only used in older versions of Kuma, I think Kong Mesh had separate files, so this file isn't shared with the Kong Mesh docs.